user-guide/Troubleshooting/Known_issues/KI_SLAutomation_memory_leak_AddScriptOutput.md

```diff
@@ -0,0 +1,22 @@
+---
+uid: KI_SLAutomation_memory_leak_AddScriptOutput
+---
+
+# SLAutomation memory leak when Engine.AddScriptOutput is used
+
+## Affected versions
+
+- DataMiner 10.1.0 and 10.2.0
+- From DataMiner 10.0.2 up to 10.3.1
+
+## Cause
+
+When the script output or subscript feature is used in an Automation script, the data that is placed in the dictionary stays in memory, causing a memory leak in SLAutomation.
```

Comment:
The issue occurs when using the script output feature with either a normal script or a subscript. The current sentence seems to suggest that the issue occurs just by using subscripts, regardless of the script output feature (which is not true).